src/components/Summary/Summary.tsx

```diff
@@ -0,0 +1,41 @@
+import { useContext } from "react";
+import { UserContext } from "../../store/user-context";
+
+import PersonalDataForm from "./PersonalDataForm";
+
+import Order from "../../models/order";
+
+import { postOrder } from "../../common/service/common-service";
+
+import { StyledSummary } from "./SummaryStyles";
+import { Button, Container } from "../../common/styles/componentsStyles";
+
+import { AddressObject } from "../../common/types/common.types";
+
+const Summary = () => {
+  const { cartItems, clearCart } = useContext(UserContext);
+
+  const totalAmount = cartItems.reduce(
+    (acc, cur) => acc + cur.amount * cur.price,
+    0
+  );
+
+  const confirmOrderHandler: (enteredAddress: AddressObject) => void = (
+    enteredAddress
+  ) => {
+    const newOrder = new Order(cartItems, enteredAddress);
```

Comment:
Is there any particular reason of creating and using the Order class? Does it do anything special under the hood? If it doesn't then why can't we just pass an object that the postOrder function expects